frontend/src/entry/settings/EntryAccessInput.tsx

```diff
@@ -28,6 +28,7 @@ export default function EntryAccessInput({ value, onUpdate, entryKind, entryRole
   const [open, setOpen] = useState(false)
   const [accessList, setAccessList] = useState<CollaboratorEntry[]>(value)
   const [userListQuery, setUserListQuery] = useState('')
+  const [manualEntityEntryErrorMessage, setManualEntityEntryErrorMessage] = useState('')
```

Comment:
This has nothing to do with entries. Consider renaming to `manualEntityInputErrorMessage`